internal/pkg/install/install.go

```diff
@@ -138,6 +149,10 @@ func Install(args string, data *userdata.UserData) (err error) {
 		return err
 	}
 
+	if err = ioutil.WriteFile(filepath.Join(constants.NewRoot, constants.BootMountPoint, "installed"), []byte{}, 0400); err != nil {
```

Comment:
I'd like to eventually write a file out with a bit of data about the specifics of the install (version, date, etc.), but this should suffice for now.